### ISSUE TITLE
Replace manual.tst with code which extracts the manuals examples dynamically

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/alnuth#8]
... so that if examples in the manual get updated, they also get tested.

This should probably wait until the manual is converted to GAPDoc and `TestDirectory` is being used.

**Comment by alex-konovalov** [comment]
@fingolfin yes, if it would be in GAPDoc, I would reuse e.g. https://github.com/gap-packages/wedderga/blob/master/makedoc.g to extract examples.
